account_financial_report_webkit/models/account_move_line.py

```diff
@@ -37,3 +37,17 @@ def _compute_last_rec_date(self):
                 move_lines = line.reconcile_partial_id.line_partial_ids
                 last_line = move_lines.sorted(lambda l: l.date)[-1]
                 line.last_rec_date = last_line.date
+
+    @api.noguess
+    def _auto_end(self, cr, context=None):
+        """
+        Create a combined index on account_move_line to handle performance
+        on balance calculating.
```

Comment:
Because I have a very poor memory myself, can you mention specifically that it is there specifically for the query that determines which balances are missing.

Reply:
Done!